pkg/daemon/ceph/osd/volume.go

```diff
@@ -145,8 +145,14 @@ func updateLVMConfig(context *clusterd.Context) error {
 	output = bytes.Replace(output, []byte("udev_rules = 1"), []byte("udev_rules = 0"), 1)
```

Comment:
@leseb can you please elaborate why we need these LVM changes when running OSDs on disks as well? Right now, we are only making these LVM changes when running OSD on PVC.

Reply:
As you can see here: https://github.com/ceph/ceph-container/blob/master/src/__DOCKERFILE_POSTINSTALL_TWEAKS__ these tweaks are also needed for bare metal containerized scenarios.

Reply:
Thanks.